### ISSUE TITLE
"Reset to factory state" option not working in menu after recent update

### ISSUE THREAD

**Issue by wachuwamekil** [ChimeraOS/chimeraos#809]
**System Information**
Systems tested: Desktop (i7 4770, RX 580, 16G Ram, 250 SSD/1TB Spindle, Asus ROG Maximus VI)
ChimeraOS Version :: 44-1
Client Version :: 2023-11-16

**Problem**
When attempting to use the Factory reset option in the client menu user is presented with an error "Failed to Initiate factory reset."

**Details**
When reviewing journlctl, command was sent to OS, but failed.  When trying to run the command manually it didn't recgonize the command.  I was not able to find the script _streamos-factory-reset-config_ anywhere on the file system, or in my path.

It does appear the script was replaced with a new method of handling this possibly?  

_New item on Holo_mirror in pkgbuild a month ago_:  https://gitlab.com/evlaV/holo-PKGBUILD/-/blob/master/steamos-reset/PKGBUILD
_New repo opened a month ago_: https://gitlab.com/evlaV/steamos-reset

Looking at the commit this appears to be a more robust factory-reset service.

**Possible Mitigation:**
Pull in older reset scripts into current ChimeraOS repo, or look to pull in the new reset services.

Screenshots below (will get logs if they are needed) ::
Error :: ![image](https://github.com/ChimeraOS/chimeraos/assets/81867187/77732c21-5166-489e-b163-aa248fb7055b)
Log Image :: https://imgur.com/a/ph2x5mU

**Comment by pastaq** [comment]
What is your expectation of what would happen when this button is pressed?

**Comment by alkazar** [comment]
This functionality was never implemented in ChimeraOS.

**Comment by al12gamer** [comment]
Just checking in here. If I'm giving away a device with ChimeraOS preinstalled, should I do a fresh install on that device to make sure everything is wiped prior to giving away?

**Comment by al12gamer** [comment]
Just asking, in the case that this button still doesn't work

**Comment by alkazar** [comment]
Yes, you should do a clean install. That functionality is not implemented.
